build.gradle

```diff
@@ -79,7 +79,7 @@ repositories {
 
 // Declare dependencies for the modules we will use.
 // A cordaProvided declaration is required for anything that we use that the Corda API provides.
-// This is required to allow us to build CorDapp modules as OSGi bundles that CPI and CPB files are built on.
+// Required to build CorDapp modules as OSGi bundles that CPI and CPB files are built on dependencies.

```

Comment:
Not a trick question, but were we asked to change this sentence structure? The original sentence works for me better I think, if you change "files are built on" to "files are then built upon".